### ISSUE TITLE
Browser Comparison not showing results

### ISSUE THREAD

**Issue by akcts** [Cognizant-Digital-Engineering-PACE/CXOptimize#8]
Hi,

We are able to see results from other tab like summary response time. Under comparison tab, the broswer comparison is not showing any results, it returns just the table columns name and the page is blank.

This is on 2.0.5 version.

Thanks,
Anand

**Comment by cognizantcxoptimize** [comment]
For transaction details to be available in browser comparison tab - you need to have same transaction sames across different browser runs.So if if named Home as transaction name in chrome web driver and the same name should be used for Firefox as well.Canb you confirm that?

**Comment by akcts** [comment]
Yes the transaction names are same, as we ran the same script on different browsers changing just the WebDriver based on the browser used.

Thanks,
Anand

**Comment by cognizantcxoptimize** [comment]
If that is the case we need to see the API component log to see if there are any exception causing the API to return no data.Also while cliclikg the browser comparison do F12 and share us the request and response from the Developer Tools
